pages/wishlist.html

```diff
@@ -5,38 +5,125 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>wishlist
-
+    <title>
+        Products
     </title>
-    <link href="../CSS/wishlist.css" rel="stylesheet" />
+    <link href="../CSS/products.css" rel="stylesheet">
     <script src="https://kit.fontawesome.com/9dead9de99.js" crossorigin="anonymous"></script>
-
-    <link href="https://ajit-tungsten-ui-dev.netlify.app/style.css" rel="stylesheet" />
+    <link href="https://ajit-tungsten-ui-dev.netlify.app/style.css" rel="stylesheet">
 </head>
 
 <body>
     <header class="header-part">
         <div class="logo">
             <h1>Lifestyle&nbsp;Store</h1>
         </div>
-        <div class="search-bar"><input type="text" placeholder="search here" /> </div>
+        <div class="search-bar">
+            <input type="text" placeholder="search here">
+        </div>
         <ul class="navlink">
             <li>
                 <a href="../index.html">Home</a>
             </li>
-
             <li>
-                <a href="cart.html"><i class="fa-solid fa-cart-arrow-down"></i></a>
+                <a href="cart.html">
+                    <i class="fa-solid fa-cart-arrow-down"></i>
+                </a>
             </li>
             <li>
-                <a href="wishlist.html"><i class="fa-solid fa-heart"></i></a>
+                <a href="wishlist.html">
+                    <i class="fa-solid fa-heart"></i>
+                </a>
             </li>
             <li>
-                <a href="login.html"><i class="fa-solid fa-user"></i></a>
+                <a href="login.html">
+                    <i class="fa-solid fa-user"></i>
+                </a>
             </li>
         </ul>
     </header>
-
+    <div class="body-container">
+        <div class="main-container">
+            <div class="product-card-container">
+                <img src="https://i.ibb.co/KL8Ly0q/3.jpg" class="product-card_container-images">
```

Comment:
you can give alt tag to images, it's a good practice.

Reply:
Ohh, thank you I missed that almost in every img tag